### ISSUE TITLE
Tests fail with -Duselongdouble

### ISSUE THREAD

**Issue by eserte** [karupanerura/Geo-Hex-V3-XS#1]
If perl is compiled with -Duselongdouble, then some tests fail:

```
#   Failed test at t/03_functions.t line 14.
#          got: '339.336963200054192'
#     expected: '339.336963200054'

#   Failed test at t/03_functions.t line 15.
#          got: '37.7041070222282436'
#     expected: '37.7041070222282'
# Looks like you failed 2 tests of 6.
t/03_functions.t .. 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/6 subtests 

#   Failed test at t/04_polygon.t line 7.
#     Structures begin differing at:
#          $got->[0]{lng} = '49.4650205761316855'
#     $expected->[0]{lng} = '49.4650205761317'
# Looks like you failed 1 test of 1.
t/04_polygon.t .... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```

(Seen on a Debian/wheezy system with perl 5.20.1)

**Comment by eserte** [comment]
Same failure happens also on a FreeBSD 10.0 system, also with a -Duselongdouble perl.

**Comment by karupanerura** [comment]
oops! thank you for reporting!
I'll fix it.

**Comment by karupanerura** [comment]
I released it as version 0.06. many thanks!
